### DISCUSSION TITLE
Web3Modal does not open mobile wallet when used on Telegram Mini App (Mobile)

### DISCUSSION THREAD

**Discussion by kang5647** [WalletConnect/walletconnect-monorepo#4590] [Developer Support]
I think a lot of people have asked about the same question. 
Is there any solution for making this work? 
![34f2f33a-96c5-4044-8be7-cda71722782d](https://github.com/WalletConnect/walletconnect-monorepo/assets/76279908/649dacf5-d5a1-4a18-af61-c08ca50ce255)

**Comment by glitch-txs** [comment]
I'll look into this
